{
  "source": "print(468 % int(input())",
  "language": "python3",
  "problem": "0003.json",
  "userId": "Shiroha_F14",
  "status": -8269,
  "subDate": "2021-05-22T07:58:09.742Z",
  "score": "100",
  "judges": [
    {
      "internalId": "8PtcD0bFImJUPSUX2Yvw5A",
      "input": 285,
      "expected": 183,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "xFONBTyj7Dmfn6W_gLIFfQ",
      "input": 199,
      "expected": 70,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "l5Ye2Up8RKgRGv1Cl2WXig",
      "input": 406,
      "expected": 62,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "LtrGtqtNfqZq7uLiUf0UqA",
      "input": 75,
      "expected": 18,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "56Yuo3wLVqV-qnDqU0n95w",
      "input": 133,
      "expected": 69,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "950qPJSGyH8Fd1wt9WOf8Q",
      "input": 144,
      "expected": 36,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "tmdo7ab7Ic3kyYo1VJ8L0Q",
      "input": 419,
      "expected": 49,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "SBgeAn1UiqFT2dPsx6W2nA",
      "input": 437,
      "expected": 31,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "SqJXrN70JLhdCSgfN3B26g",
      "input": 420,
      "expected": 48,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "iguhS7QwjoCXYvmX-EWuGQ",
      "input": 123,
      "expected": 99,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "GBkEXuhZt9D-MbR9oA6eRg",
      "input": 287,
      "expected": 181,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "4fbhAO5kHMW22pJ70dHJOg",
      "input": 310,
      "expected": 158,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "JMNQx-Kz0Woh5lWqeG5J9w",
      "input": 227,
      "expected": 14,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "OUmSuZp36un71lKGyaOQLg",
      "input": 134,
      "expected": 66,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "e_WJlG-ueeoS7dBGvc8j_A",
      "input": 357,
      "expected": 111,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "dEMPFbOFVUs6lmZ275thLg",
      "input": 152,
      "expected": 12,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "_87aQUCyMzbH_y-FvKUTWg",
      "input": 428,
      "expected": 40,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "cPBaS5FRfFJINZTn1u8pqQ",
      "input": 245,
      "expected": 223,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "vWXr5pBmxiIUmO6--W7p4A",
      "input": 363,
      "expected": 105,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "631Zf711avLUovmchV3i0Q",
      "input": 221,
      "expected": 26,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "YH3qp6zbQaHDG1yfoDAK3w",
      "input": 15,
      "expected": 3,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "PNMYL7RJE0yt5duPIUtwOA",
      "input": 233,
      "expected": 2,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "-IXJJjRWLUZM6OQH9Q0XxA",
      "input": 86,
      "expected": 38,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "PwRslyQPYwRlIodzeiBvVQ",
      "input": 279,
      "expected": 189,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "R_wDxbziBMDUpo_Hp9iE3Q",
      "input": 131,
      "expected": 75,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "7tO_VlGwDI9K0y6rZspCmw",
      "input": 134,
      "expected": 66,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "Nu1cX8FXW9IMYs4MsrN9QA",
      "input": 372,
      "expected": 96,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "Z7ZUPC5t5N46t8PNnqGm4A",
      "input": 68,
      "expected": 60,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "dC7_tSn-JQ1eob9ss6v_lw",
      "input": 408,
      "expected": 60,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "TWq324XzySKbXd8E8K-GzA",
      "input": 367,
      "expected": 101,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "U17qHCHoVByq0q0c1Uc8XA",
      "input": 429,
      "expected": 39,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "4i1gCvuRbHhJAdioNp0Ucw",
      "input": 10,
      "expected": 8,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "CYB_7kRtWc6QIf8o3cxp7Q",
      "input": 80,
      "expected": 68,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "gqq6nXh-7m5b8uIfg2Uysw",
      "input": 346,
      "expected": 122,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "_CJx7GJ4t9fZ7WSzF88gYQ",
      "input": 196,
      "expected": 76,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "6vMS3ZdjSUqJ2r6VxewjhA",
      "input": 397,
      "expected": 71,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "3D1KyrQeJ3V5AF7WyDldhQ",
      "input": 282,
      "expected": 186,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "G1sFhmcUlRhxOAo8uVtZsg",
      "input": 217,
      "expected": 34,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "f2F8_YNlUiARC2EHy-EFJw",
      "input": 249,
      "expected": 219,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    },
    {
      "internalId": "OpIxAQx1EjsOg8482QL4OQ",
      "input": 26,
      "expected": 0,
      "output": "  File \"Main.py\", line 2\n    \n                            ^\nSyntaxError: unexpected EOF while parsing\n",
      "resultText": "RE",
      "isFinished": true
    }
  ],
  "forcedExit": false
}